docs/pages/quickstart_guide.mdx

```diff
@@ -0,0 +1,78 @@
+
+# Quick Start Guide
+The Pelican platform provides a platform that makes deploying data and accessing data easy. It provides an open-source software for federating dataset repositories together and delivering objects to computing capacity. For more information on Pelican, visit the [About Pelican](./index.mdx) page.
```

Comment:
Let's go with something like:
`The developers of the Pelican Platform believe data is the lifeblood of science. That's why Pelican provides a suite of tools that make it easy for researchers to federate their data, giving them the ability to share objects from a broad range of data repositories and allowing them to connect their data with distributed computational capacity, such as the OSPool. For more information on Pelican...`